dts/bindings/mmc/renesas,rcar-emmc.yaml

```diff
@@ -0,0 +1,67 @@
+description: Renesas R-Car eMMC
+
+compatible: "renesas,rcar-mmc"
+
+include: [sdhc.yaml, mmc.yaml, pinctrl-device.yaml, reset-device.yaml]
+
+properties:
+  clocks:
+    required: true
+
+  reg:
+    required: true
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  max-frequency:
+    type: int
+    default: 200000000
+    required: true
+
+  mmc-hs200-1_8v:
+    type: boolean
+
+  mmc-hs400-1_8v:
+    type: boolean
+
+  non-removable:
```

Comment:
Please add a description to this property (although it is pretty self-explanatory, I still would like some text explaining that setting this property will affect `sdhc_card_present`)